composer.json

```diff
@@ -9,7 +9,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.4|^8.0",
```

Comment:
```suggestion
        "php": "^7.4|^8.0|^8.1",
```
Spatie's Laravel Permission support PHP 8.1

Reply:
^8.0 also covers 8.1, this line is redundant